src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/PBAPaymentController.java

```diff
@@ -108,42 +108,39 @@ public ResponseEntity<AboutToStartOrSubmitCallbackResponse> applicantOrganisatio
         validateCaseData(callbackRequest);
         final Map<String, Object> mapOfCaseData = caseDetails.getData();
 
-        if (featureToggleService.isAssignCaseAccessEnabled()) {
-            try {
-                String applicantOrgId = getApplicantOrgId(caseDetails);
-
-                if (applicantOrgId != null) {
-                    OrganisationsResponse prdOrganisation = prdOrganisationService.retrieveOrganisationsData(authToken);
-
-                    if (prdOrganisation.getOrganisationIdentifier().equals(applicantOrgId)) {
-                        log.info("Assigning case access for Case ID: {}", caseDetails.getId());
-                        try {
-                            assignCaseAccessService.assignCaseAccess(caseDetails, authToken);
-                            ccdDataStoreService.removeCreatorRole(caseDetails, authToken);
-                        } catch (Exception e) {
-                            log.error("Assigning case access threw exception for Case ID: {}, {}",
-                                caseDetails.getId(), e.getMessage());
-                            return assignCaseAccessFailure(caseDetails, emptyList());
-                        }
-                    } else {
-                        String errorMessage = "Applicant solicitor does not belong to chosen applicant organisation";
-                        log.info("{} for Case ID: {}", errorMessage, caseDetails.getId());
-                        return assignCaseAccessFailure(caseDetails, singletonList(errorMessage));
+        try {
+            String applicantOrgId = getApplicantOrgId(caseDetails);
+
+            if (applicantOrgId != null) {
+                OrganisationsResponse prdOrganisation = prdOrganisationService.retrieveOrganisationsData(authToken);
+
+                if (prdOrganisation.getOrganisationIdentifier().equals(applicantOrgId)) {
+                    log.info("Assigning case access for Case ID: {}", caseDetails.getId());
+                    try {
+                        assignCaseAccessService.assignCaseAccess(caseDetails, authToken);
+                        ccdDataStoreService.removeCreatorRole(caseDetails, authToken);
+                    } catch (Exception e) {
+                        log.error("Assigning case access threw exception for Case ID: {}, {}",
+                            caseDetails.getId(), e.getMessage());
+                        return assignCaseAccessFailure(caseDetails, emptyList());
                     }
                 } else {
-                    String errorMessage = "Applicant organisation not selected";
+                    String errorMessage = "Applicant solicitor does not belong to chosen applicant organisation";
```

Comment:
Is it okay if error log message has changed too?

Reply:
I don't know how that changed as that's not something i did. I only deleted the relevant lines and locally i don't have any history with that line being mentioned

Reply:
Ok for clarification, the error message hasn't changed. Github is reading it that way but this is not the case. It's just moved because i removed the overarching if/else statement which has cascaded down